src/Unicorn.Tests/Predicates/SerializationPresetPredicateTests.cs

```diff
@@ -37,8 +38,13 @@ public void ctor_ThrowsArgumentNullException_WhenNodeIsNull()
 		[InlineData("/implicit-nochildren/ignoredchild", false)]
 		[InlineData("/implicit-nochildren/ignored/stillignored", false)]
 		[InlineData("/implicit-nochildrenwithextrachars", false)]
-		// SOME-CHILDREN test config
-		[InlineData("/somechildren", true)]
+        // TEMPLATE-ID test config
```

Comment:
yeah, I was using my default settings with `show-whitespaces` disabled in SourceTree. Now everything should be fine.